test/new/db/tools/rabin2

```diff
@@ -104,12 +104,12 @@ EXPECT=<<EOF
 EOF
 RUN
 
-NAME=rabin2 -z pe
```

Comment:
Why did you remove this test?

Reply:
sorry, made some mistakes

Reply:
![image](https://user-images.githubusercontent.com/12135974/75498470-db591a80-59f9-11ea-8968-2201a426f2cb.png)
This was automatically generated by r2r but the test keep failing. It's '-g'.

Reply:
added it back. I really have no idea why it got deleted. I just run the test and press fix...